### PR TITLE
Improvements to Authentication guide

### PR DESCRIPTION
**Pull request by rosieyohannan**
# Description

- [x] Add OAuth flow to guide so it doesn't look like LDAP is the only option.
- [x] Improve the LDAP guide to include example entries for all settings

### CODE REVIEW

File: jekyll/_cci2/authentication.adoc [hunk 1]
```diff
@@ -5,48 +5,63 @@
...
+. Fill in the Search password field with your LDAP admin password.
+. Fill in the User search DN field with an appropriate value using the format `ou=<users>` replacing `users` with the value used in your LDAP instance.
+. Fill in the Username field with an appropriate unique identifier used for your users, for example, `mail`.
 . Fill in the Group Membership field with an appropriate value. By default, the value is `uniqueMember` for OpenLDAP and `member` for Active Directory. This field will list member `dn` for a group.
-. Fill in the Group Object Class field with an approrpiate value. By default, the value is `groupOfUniqueNames` for OpenLDAP and `group` for Active Directory. The value of the `objectClass` field indicates a `dn` is a group.
-. (Optional) Fill in the Test username and Test password fields with a test email and password for an LDAP user you want to test.
+. Fill in the Group Object Class field with an appropriate value. By default, the value is `groupOfUniqueNames` for OpenLDAP and `group` for Active Directory. The value of the `objectClass` field indicates a `dn` is a group.
```
Comment (smaant) [inline jekyll/_cci2/authentication.adoc]: This seems almost right. I would edit it probably a bit: A name of an Object Class that will identify DN as a group. Example: `groupOfUniqueNames`

File: jekyll/_cci2/authentication.adoc [hunk 2]
```diff
@@ -5,48 +5,63 @@
...
+
+Once your installation is up and running, you can simply provide users with a link to access the application - for example `<your-circleci-hostname>.com` and they will be prompted to set up and account by running through the GitHub/GitHub Enterprise OAuth flow.
+
+== LDAP
+As an alternative to the OAuth/GitHub option described above, you can use LDAP authentication. This section describes how to enable, configure, and test CircleCI to authenticate users with OpenLDAP or Active Directory credentials.
 
 NOTE: LDAP is not supported with existing installations, only clean installations may use LDAP.
 
-toc::[]
+CAUTION: Enabling LDAP will mean it is the _only_ way for users to authenticate their account for your installation.
```
[the remark this answers is not in the document]
Comment (rosieyohannan) [reply jekyll/_cci2/authentication.adoc]: Done

File: jekyll/_cci2/authentication.adoc [hunk 3]
```diff
@@ -5,48 +5,63 @@
...
 
-== Prerequisites
+=== Prerequisites
 
-* Install and configure your LDAP server and Active Directory.
-* GitHub Enterprise must be configured and is the source of organizations and projects to which users have access.
+* Install and configure your OpenLDAP server or Active Directory.
+* GitHub Enterprise must be configured and be the source of organizations and projects to which users have access.
```
[the remark this answers is not in the document]
Comment (rosieyohannan) [reply jekyll/_cci2/authentication.adoc]: Done

File: jekyll/_cci2/authentication.adoc [hunk 4]
```diff
@@ -5,48 +5,63 @@
...
-image::LDAP.png[Enable LDAP]
 . Fill in your LDAP instance Hostname and port number.
+// how do they do this?
 . Select the encryption type (plain text is not recommended).
+// change screenshot
 . Fill in the Search user field with the LDAP admin username using the format `cn=<admin>,dc=<example>,dc=<org>` replacing `admin`, `example`, and `org` with appropriate values for your datacenter.
-. Fill in the Search password field with the LDAP admin password.
-. Fill in the User search DN field with an approrpiate value using the format `ou=<users>` replacing `users` with the value used in your LDAP instance.
-. Fill in the Username field with an approriate unique identifier used for your users, for example, `mail`.
+. Fill in the Search password field with your LDAP admin password.
```
[the remark this answers is not in the document]
Comment (rosieyohannan) [reply jekyll/_cci2/authentication.adoc]: Done

File: jekyll/_cci2/authentication.adoc [hunk 5]
```diff
@@ -5,48 +5,63 @@
...
 . Log in as administrator to the Management Console for your newly installed CircleCI 2.0 instance.
 . Navigate to the Settings page and check the Enable LDAP-only Authentication button. Select either OpenLDAP or Active Directory.
-+
-.Enable LDAP
-image::LDAP.png[Enable LDAP]
 . Fill in your LDAP instance Hostname and port number.
+// how do they do this?
 . Select the encryption type (plain text is not recommended).
+// change screenshot
 . Fill in the Search user field with the LDAP admin username using the format `cn=<admin>,dc=<example>,dc=<org>` replacing `admin`, `example`, and `org` with appropriate values for your datacenter.
```
[the remark this answers is not in the document]
Comment (rosieyohannan) [reply jekyll/_cci2/authentication.adoc]: Done

File: jekyll/_cci2/authentication.adoc [hunk 6]
```diff
@@ -5,48 +5,63 @@
...
-. Fill in the Search password field with the LDAP admin password.
-. Fill in the User search DN field with an approrpiate value using the format `ou=<users>` replacing `users` with the value used in your LDAP instance.
-. Fill in the Username field with an approriate unique identifier used for your users, for example, `mail`.
+. Fill in the Search password field with your LDAP admin password.
+. Fill in the User search DN field with an appropriate value using the format `ou=<users>` replacing `users` with the value used in your LDAP instance.
```
[the remark this answers is not in the document]
Comment (rosieyohannan) [reply jekyll/_cci2/authentication.adoc]: Done

File: jekyll/_cci2/authentication.adoc [hunk 7]
```diff
@@ -5,48 +5,63 @@
...
+. Fill in the Username field with an appropriate unique identifier used for your users, for example, `mail`.
```
[the remark this answers is not in the document]
Comment (rosieyohannan) [reply jekyll/_cci2/authentication.adoc]: Done

File: jekyll/_cci2/authentication.adoc [hunk 8]
```diff
@@ -5,48 +5,63 @@
...
 . Fill in the Group Membership field with an appropriate value. By default, the value is `uniqueMember` for OpenLDAP and `member` for Active Directory. This field will list member `dn` for a group.
```
[the remark this answers is not in the document]
Comment (rosieyohannan) [reply jekyll/_cci2/authentication.adoc]: Done

File: jekyll/_cci2/authentication.adoc (hunk 1 above)
Comment (rosieyohannan) [reply jekyll/_cci2/authentication.adoc]: Done

**Comment by rosieyohannan** [comment]
@smaant or @GERey or @michelle-luna - I think I'm happy with these changes now, could one of you do a final review before merging?

**Comment by smaant** [comment]
LGTM but I would recommend getting a final 👌from George or Michelle

**Pull request merged**